dev-tools/cmd/asset/asset.go

```diff
@@ -0,0 +1,89 @@
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"go/format"
+	"io/ioutil"
+	"os"
+	"text/template"
+
+	"github.com/elastic/beats/libbeat/asset"
+)
+
+var pkg *string
+
+func init() {
+	pkg = flag.String("pkg", "", "Package name")
+}
+
+var tpl = template.Must(template.New("normalizations").Parse(`
+// ${ES_BEATS}/dev-tools/cmd/asset/asset.go
+// MACHINE GENERATED BY THE ABOVE COMMAND; DO NOT EDIT
```

Comment:
This should be changed to `Code generated by beats/dev-tools/cmd/asset/asset.go - DO NOT EDIT.` Or something that matches this pattern https://github.com/golang/go/issues/13560#issuecomment-288457920. Then tools like IntelliJ and golint will recognize it as being generated.